sql/postgres/sqlspec.go

```diff
@@ -334,6 +334,14 @@ func convertIndex(spec *sqlspec.Index, t *schema.Table) (*schema.Index, error) {
 		}
 		idx.Attrs = append(idx.Attrs, &IndexInclude{Columns: include})
 	}
+
+	if attr, ok := spec.Attr("concurrently"); ok {
+		concurrently, err := attr.Bool()
+		if err != nil {
+			return nil, err
+		}
+		idx.Attrs = append(idx.Attrs, &IndexBuildParams{Concurrently: concurrently})
+	}
```

Comment:
I suggest not adding it to the HCL schema. The reason is that spec/schema defines the _"what"_ (the schema structure in the database) and this attribute is something that presents the _"how"_ (how to apply the change).

We have a rough design of how to hook into the apply/plan step in the declarative workflow and change the "how", but since you're using Ent, this should not be a blocker for you, right? 
```suggestion
```